packages/trpc/server/routers/viewer/teams/roundRobin/roundRobinReassign.handler.ts

```diff
@@ -0,0 +1,19 @@
+import { roundRobinReassignment } from "@calcom/features/ee/round-robin/roundRobinReassignment";
+import type { TrpcSessionUser } from "@calcom/trpc/server/trpc";
+
+import type { TRoundRobinReassignInputSchema } from "./roundRobinReassign.schema";
+
+type RoundRobinReassignOptions = {
+  ctx: {
+    user: NonNullable<TrpcSessionUser>;
+  };
+  input: TRoundRobinReassignInputSchema;
+};
+
+export const roundRobinReassignHandler = async ({ input }: RoundRobinReassignOptions) => {
+  const { bookingId } = input;
+
+  await roundRobinReassignment({ bookingId });
+};
```

Comment:
should we also add a check here to verify if logged in user can access this booking id or not?